.github/workflows/i18n-consistency-checker.yaml

```diff
@@ -72,18 +61,41 @@ jobs:
           EOM
             fi
           done
+
+      - name: Check issue.md existence
```

Comment:
Not sure if a GHA is needed to check for file existence.

Maybe this can be done directly with a GHA **expression**? Could not find the syntax for that though.
Also assumed that the file-existence-action probably was created or a reason (i.e. that it was not possible differently).

Reply:
Cross post #636 here
I edited this. Please merge this 🙏